vault/build/titusvaults/sources/Vault.move

```diff
@@ -188,19 +292,19 @@ module titusvaults::Vault {
         event::emit(instant_withdraw_vault_event);
     }
 
-    public (friend) fun standard_withdraw_vault<VaultT, AssetT>(account: &signer,  amount: u64) acquires Round, Vault, VaultMap {
+    // standard withdraw vault
+    public (friend) fun standard_withdraw_vault<VaultT, AssetT>(account: &signer,  amount: u64) acquires RoundState, Vault, VaultMap {
         let user_addr = address_of(account);
 
+        let round_state = borrow_global_mut<RoundState>(@titusvaults);
         let vault = borrow_global_mut<Vault<VaultT, AssetT>>(@titusvaults);
-        let vault_map = borrow_global_mut<VaultMap<VaultT, AssetT>>(@titusvaults);
-        let user_shares = *smart_table::borrow(&vault_map.shares, user_addr);
-
-        let current_round = borrow_global<Round>(@titusvaults);
+        let vault_map = borrow_global_mut<VaultMap>(@titusvaults);
+        let user_shares = *smart_table::borrow(&round_state.shares, user_addr);
 
         let current_deposit = *smart_table::borrow(&mut vault_map.deposits, user_addr);
 
         // assert that standard withdrawal can only occur after the execution phase
-        assert!(current_round.round >= vault.creation_round + 2, E_NOT_STANDARD_WITHDRAWAL);
+        assert!(round_state.current_round_id >= vault.creation_round + 2, E_NOT_STANDARD_WITHDRAWAL);
```

Comment:
no, standard withdraw is only allowed after exercise time + exercise buffer time

Reply:
@0xriazaka 
This condition should also be based on time and not roundID